js-exercises/parse-ms/parseMs.js

```diff
@@ -0,0 +1,22 @@
+function parseMs(ms) {
```

Comment:
Variable name ms can be chosen better. It would be nice to write completely. Same goes with function name parseMs.